functions_root/react-app/files/index.jsx

```diff
@@ -0,0 +1,5 @@
+import React from './web_modules/react.js';
+import ReactDOM from './web_modules/react-dom.js';

```

Comment:
This will eventually get written in by our build system right? E.g. esbuild converts `'react'` to `'./web_modules/react.js'`

Reply:
Exactly